src/pages/Main/Main.js

```diff
@@ -1,8 +1,42 @@
-import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useEffect, useState } from 'react'
+
+import Carousel from './components/Carousel/Carousel'
+import Product from './components/Product/Product'
+import Slider from './components/Slider/Slider'
+import Banner from './components/Banner/Banner'
+import './Main.scss'
 
 const Main = () => {
-  return <h1>hello</h1>
+  const [productData, setProductData] = useState([])
+
+  useEffect(() => {
+    fetch('http://10.58.52.150:3000/products', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    })
```

Comment:
```suggestion
    fetch('http://10.58.52.150:3000/products')
```
단순 Get 요청은 이렇게 API 주소만 작성해도 됩니다!
header에 토큰이 담기거나 body값이 있을 때는 Get 요청이더라도 작성해주셔야합니다!